docs/sdk-and-tools/indices/delegators.md

```diff
@@ -0,0 +1,52 @@
+---
+id: es-index-delegators
+title: delegators
+---
+
+
+## _id
+
+
+The `_id` field of this index is composed in this way: `blake2bHash(delegatorAddress+stakingProviderAddress)`, in a base64 encoding (example: `YZNG+r3ZwFtOj0c057MnpVnXAfmSqLai15lusLWg+KM=`).
+
+## Fields
+
+
+| Field          | Description                                                                                                                          |
+|----------------|--------------------------------------------------------------------------------------------------------------------------------------|
+| address        | The address field holds the address in a bech32 encoding of the delegator.                                                           |
+| contract       | The contract field holds the address in a bech32 encoding of the staking provider contract.                                          |
```

Comment:
The contract field holds the bech32 encoded address of the staking provider contract the address delegated to.

Reply:
done